common/src/main/java/com/microsoft/identity/common/internal/platform/DevicePopManager.java

```diff
@@ -1058,7 +1095,7 @@ private KeyPair generateNewRsaKeyPair(@NonNull final Context context,
         );
     }
 
-    private void logSecureHardwareState(@NonNull final KeyPair kp) {
+    private SecureHardwareState getSecureHardwareState(@NonNull final KeyPair kp) {
         String msg;
```

Comment:
Do we need this?

Reply:
Can remove

Reply:
ad0fd74e0745a105a3cf05b6d7dcba8bbf37f79c